Automattic/sensei
# Fix resetting a quiz also resets the course start date

Related to #6064 and #6079

This PR fixes the course start date being reset upon resetting a lesson quiz. It has been reported that some users take advantage of this bug to circumvent the course access period limit.

5971065-zd-woothemes

## Testing Instructions


1. Create a course with a lesson and a quiz.
2. Allow resetting of the quiz.
3. Start the course.
4. Find the course start date by checking the `start` meta in `wp_commentmeta`.
5. Complete the quiz and reset it.
6. The `start` meta should be the same.

## Pre-Merge Checklist
